### PR TITLE
[0.18] Updating to latest strimzi and Apache Kafka 2.6

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

Fixes #

## Proposed Changes

- updates to used Strimzi

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: test/config/100-kafka-ephemeral-triple-2.6.0.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      - name: plain
+        port: 9092
+        type: internal
+        tls: false
+      - name: tls
+        port: 9093
+        type: internal
+        tls: true
+    config:
+      offsets.topic.replication.factor: 3
```
Comment (pierDipi) [inline test/config/100-kafka-ephemeral-triple-2.6.0.yaml:19]: A cluster composed of 1 replica but the offsets replication factor is 3.
Is it valid?

Comment (pierDipi) [reply test/config/100-kafka-ephemeral-triple-2.6.0.yaml:19]: `offsets.topic.replication.factor`
```
The replication factor for the offsets topic (set higher to ensure availability). Internal topic creation will fail until the cluster size meets this replication factor requirement.
```

Comment (matzew) [reply test/config/100-kafka-ephemeral-triple-2.6.0.yaml]: stupid typo - I've updated @pierDipi

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**